dh-make-pecl

```diff
@@ -12,6 +12,20 @@ PROGNAME=dh-make-pecl
 PROGVERSION=0.6.2
 PREFIX=/usr
 
+default_php_version()
+{
+    . /etc/lsb-release
+    if [ "$DISTRIB_CODENAME" = bionic ]; then
+        echo 7.2
+    elif [ "$DISTRIB_CODENAME" = xenial ]; then
+        echo 7.0
+    else
+        >&2 echo "Unknonw PHP target version.  Use '--phpversion' or export PHPVERSION"
+        exit 1
+    fi
+}
```

Comment:
This is Ubuntu-specific code, while the tool is not.

Might be better to check the default installed version of php?

```bash
function get_php_version {
    php --version | head -1 | sed 's/PHP \([0-9]\.[0-9]\).*/\1/g'
}
```